src/AdminMenus/FormAdminMenu.php

```diff
@@ -294,19 +317,44 @@ private function getNoItemsMessage(string $type): array
 					]),
 				];
 				break;
-			case 'trash':
+			case UtilsConfig::SLUG_ADMIN_LISTING_TRASH:
+				if ($parent === UtilsConfig::SLUG_ADMIN_LISTING_RESULTS) {
+					$output = [
+						Components::render('highlighted-content', [
+							'highlightedContentTitle' => \__('Trash list is empty', 'eightshift-forms'),
+							// Translators: %s is the link to the forms listing page.
+							'highlightedContentSubtitle' => \sprintf(\__('
+								Your don\'t have any result outputs in trash.<br />
+								<br /><a class="es-submit es-submit--outline" href="%s">Go to result outputs</a>', 'eightshift-forms'), UtilsGeneralHelper::getListingPageUrl(UtilsConfig::SLUG_ADMIN_LISTING_RESULTS, '', $parent)),
+							'highlightedContentIcon' => 'emptyStateTrash',
+						]),
+					];
+				} else {
+					$output = [
+						Components::render('highlighted-content', [
+							'highlightedContentTitle' => \__('Trash list is empty', 'eightshift-forms'),
+							// Translators: %s is the link to the forms listing page.
+							'highlightedContentSubtitle' => \sprintf(\__('
+								Your don\'t have any form in trash.<br />
+								<br /><a class="es-submit es-submit--outline" href="%s">Go to your forms</a>', 'eightshift-forms'), $listingUrl),
+							'highlightedContentIcon' => 'emptyStateTrash',
+						]),
+					];
+				}
+				break;
+			case UtilsConfig::SLUG_ADMIN_LISTING_RESULTS:
 				$output = [
 					Components::render('highlighted-content', [
-						'highlightedContentTitle' => \__('Trash list is empty', 'eightshift-forms'),
+						'highlightedContentTitle' => \__('Result output list is empty', 'eightshift-forms'),
 						// Translators: %s is the link to the forms listing page.
 						'highlightedContentSubtitle' => \sprintf(\__('
-							Your don\'t have any form in trash.<br />
-							<br /><a class="es-submit es-submit--outline" href="%s">Go to your forms</a>', 'eightshift-forms'), $listingUrl),
-						'highlightedContentIcon' => 'emptyStateTrash',
+							Your don\'t have any result outputs.<br />
+							<br /><a class="es-submit es-submit--outline" href="%s">Go to your forms</a>', 'eightshift-forms'), UtilsGeneralHelper::getListingPageUrl()),
```

Comment:
```suggestion
							<br /><a class="es-submit es-submit--outline" href="%s">Go to your forms</a>', 'eightshift-forms'), esc_url(UtilsGeneralHelper::getListingPageUrl())),
```

I'm not sure if these helpers are escaped or not 🤷🏼‍♂️